lib/chef/file_access_control/unix.rb

```diff
@@ -26,6 +26,16 @@ module Unix
       UINT = (1 << 32)
       UID_MAX = (1 << 32) - 10
 
+      module ClassMethods
+        def writable?(path)
+          ::File.writable?(path)
+        end
+      end
+
+      def self.included(base)
+        base.send :extend, ClassMethods
+      end
```

Comment:
i stumbled over this for a minute, you might put a comment here on what this is doing.